client/js/table-view.js

```diff
@@ -1,89 +1,182 @@
 const { getLetterRange } = require('./array-util');
 const { createTR, createTH, createTD, removeChildren } = require('./dom-util');
 class TableView{
-	constructor(model){
-		this.model = model;
-	}
-	init(){
-		this.initDomReferences();
-		this.initCurrentCell();
-		this.renderTable();
-		this.attachEventHandlers();
-	}
-	initDomReferences(){
-		this.headerRowEl = document.querySelector('THEAD TR');
-		this.sheetbodyEl = document.querySelector('TBODY');
-		this.formulabarEl = document.querySelector('#formulabar');
-	
-	}
-	initCurrentCell(){
-		this.currentCellLocation = {col:0,row:0};
-		this.renderFormulaBar();
-	}
-	normalizeValueForRendering(value){
-		return value || '';
-	}
-	renderFormulaBar(){
-		const currentCellValue = this.model.getValue(this.currentCellLocation);
-		this.formulabarEl.value = this.normalizeValueForRendering(currentCellValue);
-		this.formulabarEl.focus();
-	}
-	renderTable(){
-		this.renderTableHeader();
-		this.renderTableBody();
+  constructor(model){
+    this.model = model;
+  }
+  init(){
```

Comment:
Same here: please use one blank line in between each method in a class or function in the same file.